From f0c52798b761280583379ba267b0ecc1ea34bbfc Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Sat, 15 Jun 2019 03:16:45 -0700
Subject: [PATCH] Fix build with IGNORE_MTAB (#425)

chdir_to_parent and check_is_mount are used by should_auto_mount, added
in fuse-3.3.0, regardless of whether IGNORE_MTAB is defined.
---
 util/fusermount.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/util/fusermount.c b/util/fusermount.c
index 5e0b104..56da6ec 100644
--- a/util/fusermount.c
+++ b/util/fusermount.c
@@ -208,6 +208,7 @@ static int may_unmount(const char *mnt, int quiet)
 
 	return 0;
 }
+#endif
 
 /*
  * Check whether the file specified in "fusermount3 -u" is really a
@@ -395,6 +396,7 @@ static int chdir_to_parent(char *copy, const char **lastp)
 	return 0;
 }
 
+#ifndef IGNORE_MTAB
 /* Check whether the kernel supports UMOUNT_NOFOLLOW flag */
 static int umount_nofollow_support(void)
 {
-- 
2.20.1

